server/plugin.go

```diff
@@ -46,8 +46,9 @@ const (
 	SettingReminders              = "reminders"
 	SettingOn                     = "on"
 	SettingOff                    = "off"
+	chimeraGitLabAppIdentifier    = "plugin-gitlab"
```

Comment:
Nit: I think this value should have a newline separating it from the values above as it was before